uint8 axis
uint16 node_id
---
bool success